server/controllers/prisonerCellHistoryController.ts

```diff
@@ -144,7 +144,7 @@ export default class PrisonerCellHistoryController {
             name: formatName(occupant.firstName, '', occupant.lastName, { style: NameFormatStyle.firstLast }),
             profileUrl: `/prisoner/${occupant.offenderNo}`,
           })),
-        prisonerName: formatName(firstName, '', lastName),
+        prisonerName: formatName(firstName, '', lastName, { style: NameFormatStyle.lastCommaFirst }),
```

Comment:
Is this the right way round?
This is being set to `lastCommaFirst` but `breadcrumbPrisonerName` below is `firstLast`

Reply:
oh i didn't notice `breadcrumbPrisonerName` because it's not used in the actual page - that is the right way around though yeah